.cursor/rules.mdc

```diff
@@ -0,0 +1,54 @@
+---
+description: General rules for cursor
+alwaysApply: true
+---
+
+# Cursor Rules for Konflux Architecture Repository
```

Comment:
Need a better title here to explain what are the ruled about. Per haps these are "contribution guidelines"?